Gemfile

```diff
@@ -6,7 +6,12 @@ gemspec
 
 gem "quickdraw", git: "https://github.com/joeldrapper/quickdraw.git"
 gem "benchmark-ips"
-gem "solargraph"
-gem "rubocop"
-gem "ruby-lsp"
-gem "simplecov"
+
+if RUBY_ENGINE == "ruby"
```

Comment:
This was the only way I could get GitHub to stop installing these gems. There must be a better way to do it.

Reply:
`gem "foo", platforms: :mri` (or `platforms: [:mri, :windows]` if you care about Windows)
https://bundler.io/v2.5/man/gemfile.5.html#PLATFORMS

Reply:
I actually don't want it to install any of the development gems in CI. I'd rather find a way to exclude this whole group.

But I'll add that anyway for developers. Thanks for the tip.